src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -199,3 +199,28 @@ def upload_file_to_dsn(self, input_file, dataset_name, encoding=_ZOWE_FILES_DEFA
             response_json = self.write_to_dsn(dataset_name, file_contents)
         else:
             raise FileNotFound(input_file)
+
+    def write_to_uss(self, filepath_name, data, encoding=_ZOWE_FILES_DEFAULT_ENCODING):
+        """Write content to an existing UNIX file.
+        Returns
+        -------
+        json
+            A JSON containing the result of the operation
+        """
+        custom_args = self.create_custom_request_arguments()
+        custom_args["url"] = "{}fs/{}".format(self.request_endpoint, filepath_name)
```

Comment:
If `filepath_name` contains a leading slash, then the string generated here will contain two consecutive slashes and the z/OSMF API doesn't like that.

I believe we could fix this by removing the leading slash if present - something like `filepath_name.lstrip("/")`

Reply:
thanks !